Project.toml

```
@@ -12,11 +12,11 @@ SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-DSP = "0.5.1, 0.6"
-FFTW = "0.3, 1"
-Reexport = "0.2, 1.0"
-SpecialFunctions = "0.7.1, 0.8, 0.9, 0.10"
-julia = "1"
+DSP = "0.6"
+FFTW = "1"
+Reexport = "0.2, 1"
+SpecialFunctions = "1"
+julia = "1.5"
```

All other compat changes are unnecessary to me; dropping old versions can easily shake the entire downstream ecosystem.

```suggestion
DSP = "0.5.1, 0.6"
FFTW = "0.3, 1"
Reexport = "0.2, 1.0"
SpecialFunctions = "0.7.1, 0.8, 0.9, 0.10, 1"
julia = "1"
```